### PR TITLE
Added a trump tweet filter.

### PR DESCRIPTION
**Pull request by Bezbakri**
Dove please make sure to remove the final trump tweet. I can't do that.

### CODE REVIEW

File: src/fullyautomatednutcracker/cogs/filters.py [hunk 1]
```diff
@@ -901,6 +928,94 @@ async def commie(self, ctx):
...
+        trump_tweet = Image.new(mode, size, color)
+        
+        trump_tweet.paste(trump_tweet_header)
+        trump_tweet.paste(text_image, (42, 150))
+        trump_tweet.paste(trump_tweet_footer, (0, text_image_y_dimension+151))
+        
+        
+        trump_tweet_path = "content/trump_tweet.png"
+        
+        trump_tweet.save(trump_tweet_path)
```
Comment (dovedevic) [inline src/fullyautomatednutcracker/cogs/filters.py]: dont save anything to the system harddrive, store said file in a BytesIO object

File: src/fullyautomatednutcracker/cogs/filters.py [hunk 2]
```diff
@@ -901,6 +928,94 @@ async def commie(self, ctx):
...
+        size = (1094, text_image_y_dimension)
+        color = (255, 255, 255)
+        text_image = Image.new(mode, size, color)
+        
+        writing_text = ImageDraw.Draw(text_image)
+        writing_text.text(xy=(0, 0), text=text, font=font, fill='#000000')
+        
+        
+        
+
```
Comment (dovedevic) [inline src/fullyautomatednutcracker/cogs/filters.py]: remove more than 2x newlines please

**Review by dovedevic — CHANGES_REQUESTED** [review]

**Review by Bezbakri — COMMENTED** [review]
Fixed both. BytesIO object used.

File: src/fullyautomatednutcracker/cogs/filters.py [hunk 3]
```diff
@@ -901,7 +928,80 @@ async def commie(self, ctx):
...
+        
+        trump_tweet.paste(trump_tweet_header)
+        trump_tweet.paste(text_image, (42, 150))
+        trump_tweet.paste(trump_tweet_footer, (0, text_image_y_dimension+151))
+        
+        
+        with io.BytesIO() as image_binary:
+             trump_tweet.save(image_binary, 'PNG')
+             image_binary.seek(0)
+             await ctx.send(file=discord.File(fp=image_binary, filename='trump_says.png'))
```
Comment (dovedevic) [inline src/fullyautomatednutcracker/cogs/filters.py]: just remembered this is actually handled for you in `save_img_and_send`. Using this method allows this image to be used in chain commands, for example, `$djtj some text for the tweet`, then `$invert`

To utilize this, add the following here in-replacement of what's highlighted

````
await self.save_img_and_send(arguments, ctx.author, ctx.channel, trump_tweet, file_name="trump_says", things_to_close=(trump_tweet, trump_tweet_footer, trump_tweet_header))
````

File: src/fullyautomatednutcracker/cogs/filters.py [hunk 4]
```diff
@@ -901,7 +928,80 @@ async def commie(self, ctx):
...
-
+        
+        
+    @commands.command(
+        name = "trumptweet",
+        aliases = ['trump', 'trumpet']        
+    )
+    async def djtj(self, ctx, *, text = None):
+        """Trump tweets whatever you say."""
+
```
Comment (dovedevic) [inline src/fullyautomatednutcracker/cogs/filters.py]: Add the following here provided you want to support what was listed in the later remark:

```
arguments = await self.get_args_from_message(ctx)
if not arguments:
    return
```

Comment (Bezbakri) [reply src/fullyautomatednutcracker/cogs/filters.py]: wait, do i also replace text with arguments, or leave the rest of the function as-is?

File: src/fullyautomatednutcracker/cogs/filters.py (hunk 3 above)
Comment (Bezbakri) [reply src/fullyautomatednutcracker/cogs/filters.py]: Added that now.

**Review by dovedevic — APPROVED** [review]

**Pull request merged**